shell/key-bindings.bash

```diff
@@ -53,21 +53,16 @@ __fzf_cd__() {
 
 __fzf_history__() (
   local line
-  shopt -u nocaseglob nocasematch
   line=$(
-    HISTTIMEFORMAT= builtin history |
-    FZF_DEFAULT_OPTS="--height ${FZF_TMUX_HEIGHT:-40%} $FZF_DEFAULT_OPTS --tac --sync -n2..,.. --tiebreak=index --bind=ctrl-r:toggle-sort $FZF_CTRL_R_OPTS +m" $(__fzfcmd) |
-    command grep '^ *[0-9]') &&
-    if [[ $- =~ H ]]; then
-      sed 's/^ *\([0-9]*\)\** .*/!\1/' <<< "$line"
-    else
-      sed 's/^ *\([0-9]*\)\** *//' <<< "$line"
-    fi
+    builtin fc -lnr -2147483648 |
```

Comment:
I'm actually not really familiar with `fc` command of bash. But it seems to me that `1` instead of `-2147483648` suffices. Am I missing something?

Reply:
The problem is it doesn't work if the history list doesn't start at `1`, i.e. if you overflow `$HISTSIZE`:
```Shell
> HISTSIZE=7
> for i in {1..9}; do history -s $i; done
> fc -lnr 1
bash: fc: history specification out of range
```